packages/sdk-browser/src/sdk-client.ts

```diff
@@ -392,23 +453,29 @@ export class TurnkeyPasskeyClient extends TurnkeyBrowserClient {
       ],
     });
 
-    const readWriteSession = {
-      authBundle: authBundle,
-      sessionExpiry: Date.now() + Number(expirationSeconds) * 1000,
+    const session: ReadWriteSession = {
+      credentialBundle: authBundle,
+      expiry: Date.now() + Number(expirationSeconds) * 1000,
     };
 
-    // store auth bundle in local storage
-    await setStorageValue(StorageKeys.ReadWriteSession, readWriteSession);
+    await this.saveSession({
+      organizationId: organizationId!,
+      organizationName: "",
```

Comment:
is this (and `username`) an empty string by default?

Reply:
Okay I modified it, it is empty if it is not found in local storage